horreum-backend/src/main/resources/db/changeLog.xml

```diff
@@ -4090,4 +4090,14 @@
             CREATE TRIGGER before_variable_delete BEFORE DELETE ON variable FOR EACH ROW EXECUTE FUNCTION before_variable_delete_func();
         </sql>
     </changeSet>
+
+    <changeSet id="107" author="jwhiting">
+        <sql>
+            delete from datapoint dp where dp.dataset_id not in (select ds.id from dataset ds);
```

Comment:
i don't think we want to automate this, but handle it as a one-off manual process. It is specific to our env. adn this could break quietly future systems. @jesperpedersen thoughts?